__tests__/infinite_test.js

```diff
@@ -448,6 +448,37 @@ describe("React Infinite's Infinite Scroll Capabilities", function() {
     expect(infiniteSpy).not.toHaveBeenCalled();
   });
 
+  it('triggers the onInfiniteLoad function when containerHeight changes', function() {
```

Comment:
you may provide the case where `useWindowAsScrollContainer` is `true` so when containerHeight changes, `onInfiniteLoad` is not called